content/general/changelog.md

```diff
@@ -6,6 +6,19 @@ weight: 4
 
 Below you can find a full list of changes to VU, corresponding to every VU update, with the newest update at the top. Updates that have the `ONLY IN DEV` tag next to them are currently only available in the `dev` branch.
 
+{{% changeset 19963 "02/11/2022" true %}}
+- VeniceEXT was updated to `1.12.0` ([changelog](/vext/changelog/#1_12_0)).
+- Add [vu.DisablePreRound](/hosting/commands/#vudisablepreround) RCON command.
+- Introduce `-username` and `-password` as [client launch arguments](/general/args/#client-arguments) to automatically login.
+- Introduce `-maxPlayers` as [server launch argument](/general/args/#server-arguments). Note: Spectator slots count as well.
+- CEF got upgraded to `4638`. This will come with potential bug & crash fixes such as fixing black screen and out of memory issues. The performance and the memory usage should be overall improved.
+- CEF console warn- and error messages are now forwarded to the VU console.
```

Comment:
`warn-`. Also, it should say `WebUI` instead of `CEF`.

Reply:
I think we shouldn't mention CEF here but Chromium and it's version